src/xcode/ENA/ENA/Resources/Localization/ar.lproj/Localizable.strings

```diff
@@ -0,0 +1,900 @@
+
+"ExposureDetection_Explanation_Text_Low" = "خطر العدوى لديك منخفض نظرًا لعدم تسجيل أي تجمُّع مع شخص مصاب بمرض كوفيد-19 أو تم تقييد تجمُّعك لفترة قصيرة وعلى مسافة أكبر.\n\nوبالتالي، تم تصنيف خطر العدوى على أنه منخفض.\n\nيتم تقدير خطر العدوى من بيانات تسجيل المخالطة (المدة ومدى الاقتراب) محليًا على جهازك. ولا يمكن الاطلاع على خطر العدوى من جانب أي شخص آخر أو تمريره إليه.";
+
+"ExposureDetection_Explanation_Text_High" = "خطر العدوى لديك مرتفع نظرًا لأنك خالطت شخصًا واحدًا على الأقل مصابًا بمرض كوفيد-19، منذ يومين على مدى فترة زمنية أطول ومن مسافة قريبة جدًا.\n\nوبالتالي، تم تصنيف خطر العدوى على أنه مرتفع.\nيتم تقدير خطر العدوى من بيانات تسجيل المخالطة (المدة ومدى الاقتراب) محليًا على جهازك. ولا يمكن الاطلاع على خطر العدوى من جانب أي شخص آخر أو تمريره إليه.\nعند العودة إلى المنزل، يرجى تجنب التعامل عن قرب مع أفراد أسرتك أو عائلتك.";
+
+"ExposureDetection_Button_Enable" = "تنشيط تسجيل المخالطة";
+
+"ExposureDetection_Button_Refresh" = "تحديث";
+
+/* Exposure Detection Errors */
+"ExposureDetectionError_Alert_Title" = "حدث خطأ ما.";
+
+"ExposureDetectionError_Alert_Action_Details" = "التفاصيل";
+
+"ExposureDetectionError_Alert_Message" = "حدث خطأ ما أثناء تسجيل المخالطة.";
+
+/* Settings */
+"Settings_StatusActive" = "تشغيل";
+
+"Settings_StatusInactive" = "إيقاف تشغيل";
+
+"Settings_StatusEnable" = "تنشيط";
+
+"Settings_StatusDisable" = "إلغاء التنشيط";
+
+"Settings_KontaktProtokollStatusActive" = "نشط";
+
+"Settings_KontaktProtokollStatusInactive" = "موقوف";
+
+"Settings_Tracing_Label" = "تسجيل المخالطة";
+
+"Settings_Notification_Label" = "الإشعارات";
+
+"Settings_Reset_Label" = "إعادة تعيين التطبيق";
+
+"Settings_Tracing_Description" = "السماح بإنشاء معرفات عشوائية لمرض كوفيد-19 ومشاركتها.";
+
+"Settings_Notification_Description" = "السماح بالإشعارات التلقائية لحالة خطر مرض كوفيد-19 لديك.";
+
+"Settings_Reset_Description" = "حذف كل البيانات في التطبيق.";
+
+"Settings_NavTitle" = "الإعدادات";
+
+/* Noticiation Settings */
+"NotificationSettings_On_Title" = "هل تريد تنشيط إشعارات حالة خطر مرض كوفيد-19 الخاصة بك؟";
+
+"NotificationSettings_On_SectionTitle" = "الإعداد";
+
+"NotificationSettings_On_RiskChanges" = "تم تغيير خطر العدوى";
+
+"NotificationSettings_On_TestsStatus" = "حالة اختبار مرض كوفيد-19";
+
+"NotificationSettings_Off_SectionTitle" = "الإعداد";
+
+"NotificationSettings_Off_EnableNotifications" = "الإشعارات";
+
+"NotificationSettings_Off_StatusInactive" = "إيقاف تشغيل";
+
+"NotificationSettings_Off_InfoTitle" = "تنشيط الإشعارات";
+
+"NotificationSettings_Off_InfoDescription" = "لتنشيط الإشعارات، يجب السماح بإشعارات تطبيق Corona-Warn-App من إعدادات جهازك.";
+
+"NotificationSettings_Off_OpenSettings" = "فتح الإعدادات";
+
+"NotificationSettings_NavTitle" = "الإشعارات";
+
+"NotificationSettings_onImageDescription" = "Eine Frau erhält eine Mitteilung von ihrer Corona-Warn-App.";
```

Comment:
Intentionally or accidentally left untranslated?

Reply:
@kaluju :)?

Reply:
intentional, will be translated later on